### ISSUE TITLE
error when searching by zip

### ISSUE THREAD

**Issue by myersnews** [TheLens/land-records#100]
the gray text suggests searching for a zip, but when i do that, i get zero results.

http://vault.thelensnola.org/realestate/search?q=70126

I think this happened because I didn't wait for the autosuggest. but because there's a bit of a delay, and people don't know the autosuggest is there, many people will do exactly what i did. how should we address this?

**Comment by ThomasThoren** [comment]
You searched for "70126" as a keyword, not as the geographic area. You have to select ZIP codes and neighborhoods in the autocomplete dropdown or their own select menus if you want the geographic area.

http://vault.thelensnola.org/realestate/search?zip=70126

**Comment by myersnews** [comment]
right, i understand that. but how do we avoid this? i think many people will search without waiting for autocomplete and then searching the zip. even if the zip did come up, you may not think oh, i need to select the zip. you’d think, yes, i want a zip, and then hit enter.
--
Steve Myers
smyers@thelensnola.org
Special projects editor, The Lens
504-298-9750
@myersnews
